### DISCUSSION TITLE
How to debug `vcpkg.exe integrate install`?

### DISCUSSION THREAD

**Discussion by patrikhuber** [microsoft/vcpkg#22674] [Q&A]
Hi,

I've had it a few times now that I create a new C++ project in VS (no CMake, just a plain VS C++ console app) but including stuff from vcpkg doesn't work. Like just now, `#include <Eigen/Core>` - `fatal error C1083: Cannot open include file: 'Eigen/Core': No such file or directory`. I've re-run `vcpkg.exe integrate install` and re-opened the VS solution, but no use.
And I can confirm the files are present in `C:\Users\Patrik\vcpkg\installed\x64-windows\include\Eigen`.

What can I do to debug/fix this?

**Comment by ras0219-msft** [comment]
Thanks for posting this issue!

In Tools -> Options -> Projects and Solutions -> Build and Run, there are settings for the "MSBuild project build output verbosity". If you increase this to at least "Normal", you will see a message like:

```
Using triplet "x64-windows" from "D:\src\vcpkg\installed\x64-windows\"
```

If you increase it to "Detailed", you'll see the command lines used to compile which should include the include directive (something like `/I"D:\src\vcpkg\installed\x64-windows\include"`)

**Comment by SeanFarrow** [comment]
I am having the same issue, admittedly it is with visual studio 2019, v16.2.2.
Have some MSBuild paths changed in this release?
If I know what's changed, I'm happy to submit a PR as I have some time and need this to work for other projects.
